### PR TITLE
daemon: update: check len inside public function

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca runcom@redhat.com

### CODE REVIEW

File: daemon/update.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ func (daemon *Daemon) ContainerUpdate(name string, hostConfig *container.HostCon
 
 // ContainerUpdateCmdOnBuild updates Path and Args for the container with ID cID.
 func (daemon *Daemon) ContainerUpdateCmdOnBuild(cID string, cmd []string) error {
+	if len(cmd) <= 0 {
```
Comment (LK4D4) [inline daemon/update.go]: It's impossible for `len` to return `< 0`, let's just check `== 0`

Comment (runcom) [inline daemon/update.go]: lol I changed it just before pushing from `==` cause I wasn't sure, thx, fixed

**Comment by cpuguy83** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM

**Pull request merged**